quickstarts/microsoft.machinelearningservices/machine-learning-workspace/azuredeploy.json

```diff
@@ -0,0 +1,157 @@
+          "westus2",
+          "westeurope",
+          "usgovvirginia"
+        ],
+        "metadata": {
+          "description": "Specifies the location of the Azure Machine Learning workspace and dependent resources."
+        }
+      },
+      "hbi_workspace":{
+        "type": "string",
+        "defaultValue": "false",
+          "allowedValues": [
+            "false",
+            "true"
+          ],
+          "metadata": {
+            "description": "Specifies whether to reduce telemetry collection and enable additional encryption."
+          }
+      }
+  },
+  "variables": {
+      "tenantId": "[subscription().tenantId]",
+      "storageAccountName": "[concat('st', parameters('name'), parameters('environment'))]",
+      "keyVaultName": "[concat('kv-', parameters('name'), '-', parameters('environment'))]",
+      "applicationInsightsName": "[concat('appi-', parameters('name'), '-', parameters('environment'))]",
+      "resourceGroupName": "[resourceGroup().name]",
+      "containerRegistryName": "[concat('cr', parameters('name'), parameters('environment'))]",
+      "workspaceName": "[concat('mlw', parameters('name'), parameters('environment'))]",
+      "storageAccount": "[resourceId(variables('resourceGroupName'), 'Microsoft.Storage/storageAccounts', variables('storageAccountName'))]",
+      "keyVault": "[resourceId(variables('resourceGroupName'), 'Microsoft.KeyVault/vaults', variables('keyVaultName'))]",
+      "applicationInsights": "[resourceId(variables('resourceGroupName'), 'Microsoft.Insights/components', variables('applicationInsightsName'))]",
+      "containerRegistry": "[resourceId(variables('resourceGroupName'), 'Microsoft.ContainerRegistry/registries', variables('containerRegistryName'))]"
+  },
+  "resources": [
+      {
+          "type": "Microsoft.Storage/storageAccounts",
+          "apiVersion": "2021-01-01",
+          "name": "[variables('storageAccountName')]",
+          "location": "[parameters('location')]",
+          "sku": {
+              "name": "Standard_RAGRS"
+          },
+          "kind": "StorageV2",
+          "properties": {
+              "encryption": {
+                  "services": {
+                      "blob": {
+                          "enabled": true
+                      },
+                      "file": {
+                          "enabled": true
+                      }
+                  },
+                  "keySource": "Microsoft.Storage"
+              },
+              "supportsHttpsTrafficOnly": true
+          }
+      },
+      {
+          "type": "Microsoft.KeyVault/vaults",
+          "apiVersion": "2021-04-01-preview",
+          "name": "[variables('keyVaultName')]",
+          "location": "[parameters('location')]",
+          "properties": {
+              "tenantId": "[variables('tenantId')]",
+              "sku": {
+                  "name": "standard",
+                  "family": "A"
+              },
+              "accessPolicies": [],
```

Comment:
how is the vault used?  (no one has access to it)

Reply:
Azure ML will add access policies during provisioning.